src/less/details/ds6/details.less

```diff
@@ -0,0 +1,24 @@
+@import "../../less/ds6/variables.less";
+@import "../../less/ds6/mixins.less";
+
+summary.details__summary {
+    color: @ds6-color-b4;
+
+    &:hover {
+        color: @ds6-color-b7;
+
+        .details__icon {
```

Comment:
Duplicate body code
Two solutions, either use a mixin or something.
OR
Put them in similar selectors in the same level
&:hover .details__icon,
.details__icon {
   &:not...etc
}

I think it's slightly better to have duplicate &:hover rather than having duplicate body code (having 1 hover for color and another for this)

Reply:
There are actually 2 different icon mixins being used here, so the body is different.
```
            .icon-chevron-down-bold-blue(9px, 5.2px);
            .icon-chevron-down-bold-blue-dark(9px, 5.2px);
```

Is there a way to make the code cleaner with that?

Reply:
@seangates @ianmcburnie

Reply:
Didn't realize they were different, leave it be. :+1: